ranking/src/experiment_config.rs

```diff
@@ -0,0 +1,77 @@
+use crate::contest_config::ContestSource;
+use crate::compute_ratings::RatingSystem;
+
+#[allow(unused_imports)]
+use crate::{CodeforcesSystem, EloRSystem, TopCoderSystem, TrueSkillSPBSystem};
+
+use serde::{Deserialize};
+use std::path::Path;
+
+#[derive(Deserialize, Debug)]
+pub struct SystemParams {
+    pub method: String,
+    pub params: Vec<f64>,
+}
+
+#[derive(Deserialize, Debug)]
+pub struct ExperimentFile {
+    pub max_contests: usize,
+    pub mu_noob: f64,
+    pub sig_noob: f64,
+    pub topk: usize,
+    pub system: SystemParams,
+    pub contest_source: String,
+}
+
+pub struct Experiment {
+    pub max_contests: usize,
```

Comment:
The optional filters `max_contests` and `topk` could be `Option<usize>`

Reply:
Will keep the parameters explicit for now, since we don't have values that indicate "take all".